src/odemis/util/dataio.py

```diff
@@ -62,15 +62,19 @@ def data_to_static_streams(data):
         ci = dims.find("C")  # -1 if not found
         if (((model.MD_WL_LIST in d.metadata or
               model.MD_WL_POLYNOMIAL in d.metadata) and
-             (ci >= 0 and d.shape[ci] > 1)
-             ) or
-            (ci >= 0 and d.shape[ci] >= 5)
-           ):
-            # Spectrum: either it's obvious according to metadata, or no metadata
-            # but lots of wavelengths, so no other way to display
-            name = d.metadata.get(model.MD_DESCRIPTION, "Spectrum")
-            klass = stream.StaticSpectrumStream
-        elif model.MD_PIXEL_DUR in d.metadata and ti >= 0 and d.shape[ti] > 1:
+             (ci >= 0 and d.shape[ci] > 1)) or
+             (ci >= 0 and d.shape[ci] >= 5)):  # TODO if MD_WL_LIST but streak image in "Focus" mode
+            if ti >= 0 and d.shape[ti] > 1:
```

Comment:
We might want to reorganise this code (later) into something simpler like:
if there is a T or a C dimension (large enough), then create a StaticTemporalSpectrumStream.

That StaticTemporalSpectrumStream would be a generic stream which can handle Time and/or Spectrum.
In such case, we might be to use the multiple DataProjections to handle all the different type of projections.